src/herad.cpp

```diff
@@ -86,7 +87,10 @@ bool isHSQ(uint8_t * data, int size)
 		#endif
 		return false;
 	}
-	if ( *(uint16_t *)(data + 3) != size )
```

Comment:
@Malvineous this already invokes C++11, whether you specify it or not. C++98 does not officialy recognize `int*_t` types, so by using these, you are already implicitly relying in C++11

Reply:
With `CXXFLAGS="-std=c++98 -Wpedantic"`

    /usr/include/libbinio/binio.h:99:16: warning: ISO C++ 1998 does not support ‘long long’ [-Wlong-long]
       typedef long long Int;

Reply:
These types are definitely supported by earlier language versions, you just have to `#include <stdint.h>` for C or `#include <cstdint>` for C++ to use them.

Reply:
Again, C++98 has no notion of `<cstdint>`, and the fact that it works is more an accident than something you can rely on. Many old compilers will break when including `<cstdint>`, they just happen to work with `<stdint.h>` due to the presence of C99 headers on the system.

Furthermore, `long long` is not defined in C++98. C99 introduced `long long` and C++11 references C99 as a base C standard. What I'm trying to get at here is that you're using C++11 (and C99) features implicitly already.